packages/web-components/src/components/carousel/carousel.ts

```diff
@@ -722,7 +722,8 @@ class DDSCarousel extends HostListenerMixin(StableSelectorMixin(LitElement)) {
           )}">
           <div
             class="${scrollContentsClasses}"
-            style="left:${(-start * (contentsBaseWidth + gap)) / pageSize}px">
+            style="inset-inline-start:${(-start * (contentsBaseWidth + gap)) /
+            pageSize}px">
```

Comment:
Setting styles inline like this has caused problems with content security policies in the past. Can we move these styles to the `updated` hook and set them with using the CSSOM?